recipe/meta.yaml

```diff
@@ -54,26 +54,23 @@ outputs:
 
   - name: psycopg
     build:
+      number: 1
```

Comment:
You can't just bump the build for this package.  It has to be for all.  So the build bump needs to be up above, not here.

Reply:
OH, right, we have a pining here